docs/src/_reference/command-line-interface.md

```diff
@@ -732,11 +731,69 @@ meltano job remove <job_name>
 
 ### Examples
 
+##### Tasks and Blocks
```

Comment:
Can we move this explanation of what the `--tasks` argument expects up into "How to use"? It's not just example code

Reply:
I think what's missing here is an explanation of why someone would want to use blocks, and why tap/target need to go together, but e.g. dbt or great_expectations are separate.

Note that I'm not sure they need to know about this level of detail at all, since it seems below that --tasks "tap-gitlab target-postgres dbt:run" also just works. If that's the case, can we simplify the docs by not explicitly explaining blocks, since we also don't in the meltano run docs?

Reply:
This is a great question and I'd be interested in @pandemicsyn's thoughts on this. The only benefit of splitting things out into multiple array elements/lines in `meltano.yml` that I can think of is readability and ease of editing in the `meltano.yml`. Since (unless I'm grossly misunderstanding how this works--which is definitely possible) everything is run serially and any `tap ... target` subset is going to be parsed as a block anyways.

Reply:
I *think* my larger answers down below covers this. But tl;dr multiple tasks map to multiple tasks in airflow.